src/CoreWCF.Primitives/src/CoreWCF/Description/CustomEndpointAddressForMetadataBehavior.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.ObjectModel;
+using CoreWCF.Channels;
+
+namespace CoreWCF.Description
+{
+    public class CustomEndpointAddressForMetadataBehavior : IServiceBehavior
+    {
```

Comment:
I just realized, we fixed the name of the interface, but this class needs a better name. I think MetadataEndpointAddressServiceBehavior is a good name.

Reply:
I pushed the renaming of the file and also align filename with type names from previous rename.